addons/hr_recruitment/views/hr_applicant_views.xml

```diff
@@ -393,8 +393,8 @@
                 <templates>
                     <div t-name="activity-box">
                         <div>
```

Comment:
even though there is no image, spacing here would improve the feeling a little bit too